util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -1366,7 +1366,7 @@ static bool DRBG(const Span<const uint8_t> args[], ReplyCallback write_reply) {
 
   uint32_t out_len;
   if (out_len_bytes.size() != sizeof(out_len) ||
-      entropy.size() != CTR_DRBG_ENTROPY_LEN ||
+      entropy.size() != CTR_DRBG_AES_256_ENTROPY_LEN ||
```

Comment:
Will the JSON object in [modulewrapper.cc#L433](https://github.com/awslabs/aws-lc/blob/84966d22d0294b17f6037cd26f221859d363fd21/util/fipstools/acvp/modulewrapper/modulewrapper.cc#L433) be changed to include CTR-DRBG with AES-128? Will the ACVP and CAVP tests include it?

Reply:
I tried addressing this in the call-outs:
> The ACVP test suites that runs as part of unit tests is not extended to cover 128-bit keys. The reasons is: Numerous KAT tests were implemented in #156 which is the same kind of tests performed by ACVP.
See CryptoAlg-729 for backlog task to extend ACVP with 128-bit key test vectors.

Previous PR implemented KATs but not in the context of ACVP. CryptoAlg-790 will cover extending the ACVP tool. For now, I think it is out-of-scope for this line of work.

Reply:
Oh, sorry, I missed to read the PR message and dove right into the review. Thanks for the clarification.
In my opinion, I believe these changes would be needed before merging into main.